apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/BFD-766-fix-beneficiary-monthly-data/05_process_ccw_table.sql

```diff
@@ -90,256 +90,324 @@ DECLARE
+                    rcd.PTD_PBP_SEPT_ID,
+                    rcd.PTD_SGMT_SEPT_ID,
+                    rcd.RDS_SEPT_IND,
+                    rcd.CST_SHR_GRP_SEPT_CD,
+                    rcd.META_DUAL_ELGBL_STUS_SEPT_CD);
+            
+                -- Oct
+                call update_bene_monthly(
+                    rcd.BENE_ID,
+                    Oct1,
+                    rcd.FIPS_STATE_CNTY_OCT_CD,
+                    rcd.MDCR_STUS_OCT_CD,
+                    rcd.MDCR_ENTLMT_BUYIN_10_IND,
+                    rcd.HMO_10_IND,
+                    rcd.PTC_CNTRCT_OCT_ID,
+                    rcd.PTC_PBP_OCT_ID,
+                    rcd.PTC_PLAN_TYPE_OCT_CD,
+                    rcd.PTD_CNTRCT_OCT_ID,
+                    rcd.PTD_PBP_OCT_ID,
+                    rcd.PTD_SGMT_OCT_ID,
+                    rcd.RDS_OCT_IND,
+                    rcd.CST_SHR_GRP_OCT_CD,
+                    rcd.META_DUAL_ELGBL_STUS_OCT_CD);
+            
+                -- Nov
+                call update_bene_monthly(
+                    rcd.BENE_ID,
+                    Nov1,
+                    rcd.FIPS_STATE_CNTY_NOV_CD,
+                    rcd.MDCR_STUS_NOV_CD,
+                    rcd.MDCR_ENTLMT_BUYIN_11_IND,
+                    rcd.HMO_11_IND,
+                    rcd.PTC_CNTRCT_NOV_ID,
+                    rcd.PTC_PBP_NOV_ID,
+                    rcd.PTC_PLAN_TYPE_NOV_CD,
+                    rcd.PTD_CNTRCT_NOV_ID,
+                    rcd.PTD_PBP_NOV_ID,
+                    rcd.PTD_SGMT_NOV_ID,
+                    rcd.RDS_NOV_IND,
+                    rcd.CST_SHR_GRP_NOV_CD,
+                    rcd.META_DUAL_ELGBL_STUS_NOV_CD);
+            
+                -- Dec
+                call update_bene_monthly(
+                    rcd.BENE_ID,
+                    Dec1,
+                    rcd.FIPS_STATE_CNTY_DEC_CD,
+                    rcd.MDCR_STUS_DEC_CD,
+                    rcd.MDCR_ENTLMT_BUYIN_12_IND,
+                    rcd.HMO_12_IND,
+                    rcd.PTC_CNTRCT_DEC_ID,
+                    rcd.PTC_PBP_DEC_ID,
+                    rcd.PTC_PLAN_TYPE_DEC_CD,
+                    rcd.PTD_CNTRCT_DEC_ID,
+                    rcd.PTD_PBP_DEC_ID,
+                    rcd.PTD_SGMT_DEC_ID,
+                    rcd.RDS_DEC_IND,
+                    rcd.CST_SHR_GRP_DEC_CD,
+                    rcd.META_DUAL_ELGBL_STUS_DEC_CD);
+            
+                rcd_cnt  := rcd_cnt + 1;
+                loop_cnt := loop_cnt + 1;
+        
+                if loop_cnt = MAX_INNER
+                THEN
+                    EXIT inner;
+                END IF;
+                
+            END LOOP;   -- end <<inner>>
+
+            RAISE INFO 'implied COMMIT...Transaction ID: %', TXID_CURRENT();
+        
+            IF rcd_cnt % 20000 = 0
             THEN
-                RAISE INFO 'Record Count: % ...', rcd_cnt;
+                RAISE INFO 'Record Count: % ...%', rcd_cnt, NOW();
             END IF;
```

Comment:
Was this too noisy to have run every time it exits the inner loop? Right now it's out of step with `MAX_INNER`.

Reply:
Yea...the inner loop is keeping the transaction size down, while the outer loop is kinda proving a heartbeat/progress. So with 130+ M rcds, the log would be pretty noisy! Hence the discrepancy between the 10k transaction and the 20k progress.